crossbuilder

```diff
@@ -886,8 +886,9 @@ deploy_deb () {
         fi;
 
         device exec "SUDO_ASKPASS=/tmp/askpass.sh sudo -A sed -i '/Pin-Priority/c\Pin-Priority: 50' /etc/apt/preferences.d/extra-ppas.pref"
-        device exec SUDO_ASKPASS=/tmp/askpass.sh sudo -A  apt-get update -o Dir::Etc::sourcelist="/tmp/repo/sources.list" -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
-        device exec SUDO_ASKPASS=/tmp/askpass.sh sudo -A apt-get dist-upgrade -o Dir::Etc::sourcelist="/tmp/repo/all.list" --yes --force-yes
+        device exec SUDO_ASKPASS=/tmp/askpass.sh sudo -A umount /usr/share/session-migration/scripts
```

Comment:
Well... the path(s) to unmount will vary from device to device and from package to package. Not every device will have `/usr/share/session-migration/scripts` as a bind-mount. Hard-coding this here isn't particularly a good solution.

I don't have a good solution for this yet, so for now you'll have to manually run `umount` command as it dpkg complains. We probably should document this somewhere though...